tsconfig.json

```diff
@@ -22,7 +22,8 @@
 			"@build-libs/*": ["../build-libs/*"],
 			"@components/*": ["components/*"]
 		},
-		"types": ["vitest/globals", "@testing-library/jest-dom"]
+		"types": ["vitest/globals", "@testing-library/jest-dom"],
+		"strictNullChecks": false
```

Comment:
This was required to get vercel to build for me, or else it was injecting `"strictNullChecks": true` 😢